### DISCUSSION TITLE
One-time payment is stuck

### DISCUSSION THREAD

**Discussion by mgks** [community/community#52906] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

A sponsor recently tried to make a one-time donation to me and then decided to go with a monthly sponsorship also (did not cancel the one-time payment), but what happened is, his one-time payment vanished from my payouts (earlier it was visible) but still shows in his `Billing and plans > Plans and usage > Current sponsorships`, he's not charged yet and has no option to look for any more details whether he is going to be charged or not.

His Billing Summary shows only the monthly sponsorship amount in next payment, same is visible to me. Just to check, he tried making one-time payment again with one same as previous and one different amount, both of them showed error "Could not purchase this item: already has an active subscription for `__amount__` one time". If yes, it is "active", then why the user is not able to see it in the billing cycle except one place where he can't interact with it or get more details on it? and why can't he make another one-time payment? Also, why do one-time sponsorships go through billing cycles (as mentioned in #43311 - must read) instead of going directly?

There is still a lack of documentation for why this is happening and the conditions related to one-time sponsors are not very clear either. Unless there is a clear explanation to why this is happening, it will look like a technical issue.

**Comment by DHayes10** [comment]
Hi [mgks](https://github.com/mgks), 

I am sorry to hear you're having some difficulties receiving a one-time payment for your Sponsors account. If you could confirm the handle of the funder who is experiencing the error when trying to make the payment? I would be happy to research this with my billing team. Thanks!

**Comment by PratyushGhatak** [comment]
Hi I am new to Sponsor. I tried to sponsor  @getify with a one time payment of $50. It shows me that it was successful, however the amount was not deducted on my end.

Please tell me how much time does it take to process the payment?

**Comment by Wyphorn** [comment]
I also did some one-time sponsorship and after few days I saw the benefits to send more cash to the creator. But I cannot since "while your previous one-time payment is still processing.". Why should I wait for finishing the first payment before I can send a new one? Both are not related to each other. Maybe just warn here and give user to confirm a new payment, but why block with error?
